new/LIT7128QeneOzyan.xml

```diff
@@ -0,0 +1,81 @@
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʿOzyān wadqa ʾǝm-manbara tadlā... (Qǝne possibly of kwǝllǝkǝmu-type)</title>
+                <title xml:lang="en" corresp="#t1">Uzziah fell from the thrown of glory…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne of the kwǝllǝkǝmu type according to the classification in 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl> with an allusion to the fall of 
+                    Uzziah as described in <ref type="work" corresp="LIT3500Chroni">2 Chr. 26</ref> and other biblical books. 
+                    The type and author are not given in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Kwellekomu"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-11-01">Created entity.</change>
+            <change who="CH" when="2024-11-01">Updated edition with several readings supplied by Nafisa Valieva.</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55vc. Division into verses is indicated with ። . 
+                    The end of the poem is indicated with ። ። . Verse markers in <ref target="l2"/> and <ref target="l2"/> are not visible because of 
```

Comment:
It should be `<ref target="l2"/>` and `<ref target="l6"/>`, right?

Reply:
Yes, thanks!